### ISSUE TITLE
Several Broken Recipes

### ISSUE THREAD

**Issue by SynfulChaot** [stal111/Forbidden-Arcanus#410]
**Describe the bug**
The following recipes are broken:
- forbidden_arcanus:aurum_chest_boat
- forbidden_arcanus:aurum_fence
- forbidden_arcanus:aurum_fence_gate
- forbidden_arcanus:fungyss_fence
- forbidden_arcanus:fungyss_fence_gate
- forbidden_arcanus:edelwood_chest_boat
- forbidden_arcanus:edelwood_fence
- forbidden_arcanus:edelwood_fence_gate

**To Reproduce**
1. Start up Minecraft.
2. Attempt to look up the recipe for the affected items.
3. Find no recipe

**Expected behavior**
Items should be craftable.

**Error Log**
```
[22:40:24] [Server thread/ERROR] [minecraft/RecipeManager]:
 Parsing error loading recipe forbidden_arcanus:aurum_chest_boat
com.google.gson.JsonParseException: An ingredient entry needs either a tag or an item
	at net.minecraft.world.item.crafting.Ingredient.m_43919_(Ingredient.java:226) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:zeta_forge.mixins.json:IngredientMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.item.AccessIngredient,pl:mixin:APP:crafttweaker.mixins.json:common.transform.item.MixinIngredient,pl:mixin:A}
	at net.minecraftforge.common.crafting.VanillaIngredientSerializer.parse(VanillaIngredientSerializer.java:27) ~[forge-1.20.1-47.1.103-universal.jar%23297!/:?] {re:classloading}
	at net.minecraftforge.common.crafting.CraftingHelper.getIngredient(CraftingHelper.java:148) ~[forge-1.20.1-47.1.103-universal.jar%23297!/:?] {re:mixin,re:classloading}
	at net.minecraft.world.item.crafting.Ingredient.m_288218_(Ingredient.java:194) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:zeta_forge.mixins.json:IngredientMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.item.AccessIngredient,pl:mixin:APP:crafttweaker.mixins.json:common.transform.item.MixinIngredient,pl:mixin:A}
	at net.minecraft.world.item.crafting.ShapelessRecipe$Serializer.m_44275_(ShapelessRecipe.java:104) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:classloading}
	at net.minecraft.world.item.crafting.ShapelessRecipe$Serializer.m_6729_(ShapelessRecipe.java:89) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:classloading}
	at net.minecraft.world.item.crafting.ShapelessRecipe$Serializer.m_6729_(ShapelessRecipe.java:84) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:classloading}
	at net.minecraftforge.common.extensions.IForgeRecipeSerializer.fromJson(IForgeRecipeSerializer.java:23) ~[forge-1.20.1-47.1.103-universal.jar%23297!/:?] {re:classloading}
	at net.minecraft.world.item.crafting.RecipeManager.fromJson(RecipeManager.java:171) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:67) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,re:classloading,pl:mixin:APP:moonlight.mixins.json:ConditionHackMixin,pl:mixin:A}
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?] {}
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,re:classloading,pl:mixin:APP:blueprint.mixins.json:SimpleReloadInstanceMixin,pl:mixin:A}
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:classloading,re:mixin}
	at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:770) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:161) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:753) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:747) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.util.thread.BlockableEventLoop.m_18701_(BlockableEventLoop.java:139) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_129861_(MinecraftServer.java:1345) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.commands.ReloadCommand.m_138235_(ReloadCommand.java:22) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:classloading}
	at net.minecraft.server.commands.ReloadCommand.m_287964_(ReloadCommand.java:53) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:classloading}
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264) ~[brigadier-1.1.8.jar%23135!/:?] {}
	at net.minecraft.commands.Commands.m_242674_(Commands.java:257) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:classloading}
	at net.minecraft.server.network.ServerGamePacketListenerImpl.m_246958_(ServerGamePacketListenerImpl.java:1246) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:immersiveengineering.mixins.json:accessors.ServerPlayNetHandlerAccess,pl:mixin:APP:supplementaries-common.mixins.json:ServerGamePacketListenerMixin,pl:mixin:A}
	at net.minecraft.server.network.ServerGamePacketListenerImpl.m_244885_(ServerGamePacketListenerImpl.java:1223) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:immersiveengineering.mixins.json:accessors.ServerPlayNetHandlerAccess,pl:mixin:APP:supplementaries-common.mixins.json:ServerGamePacketListenerMixin,pl:mixin:A}
	at net.minecraft.util.thread.BlockableEventLoop.m_18693_(BlockableEventLoop.java:67) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source) ~[?:?] {}
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:classloading,re:mixin}
	at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:770) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:161) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:753) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:747) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.util.thread.BlockableEventLoop.m_18701_(BlockableEventLoop.java:139) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServer.java:733) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:665) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[client-1.20.1-20230612.114412-srg.jar%23292!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at java.lang.Thread.run(Unknown Source) ~[?:?] {}
```

**Versions**
 - Forbidden & Arcanus:  `1.20.1-2.2.6`
 - Valhelsia Core: `1.20.1-1.1.2`
 - NeoForge: `1.20.1-47.1.103`

**Additional context**
This error did not occur with Valhelsia Core `1.20.1-1.1.1` and Forbidden & Arcanus `1.20.1-2.2.4` or `1.20.1-2.2.5`.  It doesn't appear to be recipe related as the recipe JSON didn't change between the two versions.  I _did_ note, however, that these are recipes that contain the following unique ingredient format:
```
{
      "fabric_value": {
        "tag": "c:chests"
      },
      "forge_value": {
        "tag": "forge:chests/wooden"
      }
    }
```

I'm guessing that this ingredient format is provided by Valhelsia Core and it is the cause of the error.

**Comment by HeroBrianMack** [comment]
Hopefully this pull request can help! :)

**Comment by stal111** [comment]
Fixing this in 1.20.1 would require large rewrites to the datagen we use, so this is not happening. However, all these recipe issues have been fixed in the 1.21 version of the mod.
